docs/managing_translations_i18n.md

```diff
@@ -19,4 +19,6 @@ Decidim uses [Crowdin](https://crowdin.com/) to manage the translations.
 
 ## Test the new language
 
-To test the new language in a local development app, first, you need to add the new language to the [initializer template](https://github.com/decidim/decidim/blob/master/lib/generators/decidim/templates/initializer.rb#L9). After that, [create the `development_app` and start the server](https://github.com/decidim/decidim#browse-decidim).
+- Generate the development app and cd into it.
```

Comment:
Hi @mllocs! Can you wrap "cd" with ` so it looks like code? I know I'm being picky here...

Reply:
👍